### ISSUE TITLE
Cannot Authenticate App

### ISSUE THREAD

**Issue by cjhaviland** [LandonPatmore/yahoo-fantasy-bot#80]
Looks like Yahoo changed some stuff around compared to the current setup guide. There are some extra options when choosing what your app has access to. 

Either way when I go to authenticate I am getting an error that says, "Uh Oh! Looks like something went wrong!" This is after navigating to the Yahoo Auth page and clicking on 'Agree'.

**Comment by LandonPatmore** [comment]
I just went through creating a new application myself and there is no difference in the setup process. Sure there are more options, however the options needed to setup the bot are still there.

Others have had this same issue, are you sure that set it up EXACTLY the way it says too in the guide? Others have realized they haven't, deleted the app on yahoo, and then set it up again and it works because they missed a step.

This issue from last week had the same problem as you and they did something out of order: https://github.com/LandonPatmore/yahoo-fantasy-bot/issues/79

**Comment by hagel05** [comment]
I'm having a similar issue.  I tried using an old application that I set up and setting up from using the guide.  When creating an app now you have this form:

![image](https://user-images.githubusercontent.com/20971935/113348681-870b2300-92fc-11eb-81af-2f59d908846c.png)

**Comment by LandonPatmore** [comment]
Ah I see it now, the application type is missing. I'll have to look at this more in-depth. I'm sure it doesn't effect anything, but I'll spin up a new bot tonight and look at this.

**Comment by hagel05** [comment]
Nothing jumped out from logging.  I haven't looked at the code yet, but if I get time this afternoon I will.

**Comment by LandonPatmore** [comment]
So I just got it to work with the way the guide tells you to do it. Even though Yahoo removed the "application type", it doesn't effect anything. I actually messed up myself before I fixed it while looking at this part of step 3:

"The Redirect URI will be https://<the name of your application>.herokuapp.com/auth"

Remember the **/auth** of the URL. I forgot it, and I got the error you are seeing. Once I added it back, it worked.

I am going to close this because this is not an issue, but rather a config issue.
